packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -209,6 +225,11 @@ export class DdSdkReactNativeConfiguration {
      */
     public uploadFrequency: UploadFrequency = DEFAULTS.uploadFrequency;
 
+    /**
+     * Sets the preferred size for uploaded batches of data.
```

Comment:
I would suggest to have a bit more explanation on this parameter, because currently it is not obvious what does it change (yes, size is different, but what is the real impact?).

Android SDK says the following: `Defines the batch size (impacts the size and number of requests performed by Datadog).`, so at least we can hint the user that number of requests will be different, resulting in the different number of radio invocations (and smaller size has better chances to squeeze for the cost of more radio invocations).

iOS documentation says the following:

```
Defines the Datadog SDK policy when batching data together before uploading it to Datadog servers.
Smaller batches mean smaller but more network requests, whereas larger batches mean fewer but larger network requests.
```